thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -94,7 +94,7 @@ export function enhanceAnomalies(rawAnomalies, severityScores) {
       isUserReported: anomaly.anomalyResultSource === 'USER_LABELED_ANOMALY',
       startDateStr: moment(anomaly.anomalyStart).format('MMM D, hh:mm A'),
       durationStr: noZeroDurationArr.join(', '),
-      severityScore: score ? score.toFixed(2) : 'N/A',
+      severityScore: score && !isNaN(score) ? score.toFixed(2) : 'N/A',
```

Comment:
This fixes an edge case where score is 'NaN'

Reply:
let's use`Number.isNaN`

### documentation: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

Reply:
sounds good.